actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -123,10 +123,6 @@ def _set_rendered_content_type(format) # :nodoc:
     # :api: private
     def _normalize_render(*args, &block)
       options = _normalize_args(*args, &block)
-      #TODO: remove defined? when we restore AP <=> AV dependency
-      if defined?(request) && request.variant.present?
-        options[:variant] = request.variant
-      end
       _normalize_options(options)
       options
     end
```

Comment:
Agree – @rafaelfranca is calling out that Mailers don't know about `request`.

Other controllers can work with format, locale, etc. Why not variant? There's the underlying issue to address. Moving the method elsewhere kinda-solves one problem but creates another.